coordinators/src/main/java/com/squareup/coordinators/Coordinators.java

```diff
@@ -38,8 +38,8 @@ public static void bind(View view, CoordinatorProvider provider) {
     View.OnAttachStateChangeListener binding = new Binding(coordinator, view);
     view.addOnAttachStateChangeListener(binding);
     // Sometimes we missed the first attach because the child's already been added.
-    // Sometimes we didn't, which is why the binding keeps track and avoids double
-    // attachment.
+    // Sometimes we didn't. The binding keeps track to avoid double attachment of the Coordinator,
+    // and to guard against attachment to two different views simultaneously.
```

Comment:
Have you documented what happens if a binder is attached to a new view while already attached to another?